ui/src/lib/store/podSlice.tsx

```diff
@@ -91,6 +98,20 @@ export const createPodSlice: StateCreator<MyState, [], [], PodSlice> = (
       // @ts-ignore
       "setPodContent"
     ),
+  setPodRichContent: ({ id, richContent }) =>
+    set(
+      produce((state) => {
+        let pod = state.pods[id];
+        if (pod.type != "RICH") {
+          return;
+        }
+        pod.richContent = richContent;
+        pod.dirty = true;
```

Comment:
No need to set `pod.dirty=true` here, because `pod.richContent` doesn't need to be saved to DB. Also, this doesn't change the behavior, as `pod.dirty` is already set to `true` in `setPodContent`.

Reply:
Updated in [1ceac12](https://github.com/codepod-io/codepod/pull/343/commits/1ceac12c6675a6238ecc010e12845ee240495e9d)